templates/vue/README.md

```diff
@@ -33,3 +33,37 @@ You need to adjust your git config so the automatic linting process works correc
 ```
 git config core.hooksPath .githooks
 ```
+
+## Testing Setup
+
+The testing framework that we use, Cypress, needs to be configured with some environment variables. In the `cypress.json` file, you should see something like:
+
+```json
+{
+    "env": {
+        "DEV_URL": "http://localhost:8888",
+        "TEST_TAPESTRY_NAME": "testing",
+        "USER_TAPESTRY_NAME": "testing-user",
+        "ADMIN_USERNAME": "admin",
+        "ADMIN_PASSWORD": "Tapestry123!",
+        "SUBSCRIBER_USERNAME": "narendras",
+        "SUBSCRIBER_PASSWORD": ")%AbNZgppS)uNvxT5)OdgzhX"
```

Comment:
Can you use admin_user / admin_pass and subscriber_user / subscriber_pass instead of the values you have set here as examples, please?